managed/src/main/java/com/yugabyte/yw/controllers/TablesController.java

```diff
@@ -250,10 +235,7 @@ public Result universeList(UUID customerUUID, UUID universeUUID) {
       }
       return ok(resultNode);
     } catch (Exception e) {
-      LOG.error("Failed to get list of tables in universe " + universeUUID, e);
-      return ApiResponse.error(INTERNAL_SERVER_ERROR, e.getMessage());
-    } finally {
-      ybService.closeClient(client, masterAddresses);
```

Comment:
You do need finally block that closes the client.

Reply:
So just do try {} finally {}  ... no need to catch

Reply:
Done.

Reply:
We would need to handle the Exception
client.getTablesList() - this is throwing exception explicitly in the method -> line no 228

Reply:
I wish that client threw more meaningful exceptions. But thats outside our cleanup. Just use try-catch-finally for that part of the code then. Extract method and call that:
```
private ListTablesResponse listTables() {
YBClient client = null;    
try {
      client = ybService.getClient(masterAddresses, certificate);
      ListTablesResponse response = client.getTablesList();
} catch (Exception e) {
      throw new YWServiceException(INTERNAL_SERVER_ERROR, e.getMessage());
} finally {
ybService.closeClient(client, masterAddresses);
}
}
```

Reply:
Done. Thanks!